### DISCUSSION TITLE
Unused WbSolidReference::isClosedLoop() ?

### DISCUSSION THREAD

**Discussion by ShuffleWire** [cyberbotics/webots#4056] [Q&A]
I might be crazy, but I could not find any usage of the method WbSolidReference::isClosedLoop()
A log purposely added in it never show up neither...
I'm missing something huge ? :/ Excuse my partial C++ understanding, maybe their is some sort of inherited call or something...

**Comment by omichel** [comment]
You are right, we could probably get rid of this method. Feel free to open a new PR to do so.

**Reply by ShuffleWire** [reply]
Before doing so, is it normal that we don't need to check for loop in the references ?

**Reply by omichel** [reply]
I don't know, probably we don't need it (anymore).
Anyway, a method that is not used should be removed.
